### DISCUSSION TITLE
Can't create TCP/IP socket (10106) when trying to connect

### DISCUSSION THREAD

**Discussion by khalidJA** [spatie/laravel-backup#1106] [Q&A]
Hello
When I want to backup via command line everything works fine via command
`php artisan backup:run`
or
`php artisan backup:run --only-db`
or
`php artisan backup:run --only-files`
But when I execute the command from the controller, I have the following error:
`**Exception message: The dump process failed with exitcode 2 : Misuse of shell builtins : mysqldump: Got error: 2004: Can't create TCP/IP socket (10106) when trying to connect**`
note: this error not shown with `php artisan backup:run --only-files` command
please any help!!

**Comment by PhHitachi** [comment]
any solution here?

**Reply by deesynertz** [reply]
i have the same error happen to me today

**Comment by deesynertz** [comment]
if you fixed that error help me please

**Comment by web-dev-nav** [comment]
sammeee! for god sake,,,,please help someone:(

**Reply by erikn69** [reply]
Laravel Backup 6.x are not supported
